typescript/sdk/src/consts/chainMetadata.ts

```diff
@@ -38,21 +42,22 @@ export interface ChainMetadata {
   blockExplorers: Array<{
     name: string;
     url: string;
-    family: ExplorerFamily;
+    family?: ExplorerFamily;
     apiUrl?: string;
   }>;
   blocks: {
-    // Number of blocks to wait before considering a transaction confirmed
+    /** Number of blocks to wait before considering a transaction confirmed */
```

Comment:
Can we make `blocks` optional?

Reply:
Or in general more of these options optional like `nativeToken` or `displayShortName`

Reply:
displayShortName is already optional. nativeToken is required by the token price getter.
Blocks.confirmation is required for PI chains to be substitutable for core chains. And the timeline widget needs blocks.estimateTIme

Reply:
But thinking about this more, I will make estimateTIme optional for now

Reply:
What do you mean by 

> substitutable for core chains

?

I think its fine to make it optional and assume let's say 1 imo, this is not used for validators. In general, I don't think MP should force consumers of MP to provide configuration for all possible uses of MP itself.

I.e. if I just want to deploy a warp route but don't care about the timeline widget, I don't think we should make me add this information (similar to let's say block explorer urls).